.github/workflows/release.yml

```diff
@@ -44,3 +44,27 @@ jobs:
     - uses: actions/checkout@master
     - name: Update new version in krew-index
       uses: rajatjindal/krew-release-bot@v0.0.40
+  publish-image:
+    name: publish images
```

Comment:
```suggestion
  release-image:
    name: release image
```

Reply:
should it always be "publish a release"?

Reply:
?
This workflow takes the responsibility of releasing the images when we cut a release.

Reply:
forget it, doesn't matter, just focused on "release" should be a noun or a verb